### PR TITLE
New package: craftablescience.VPKEdit version 4.0.0

### PR DESCRIPTION
**Pull request by craftablescience**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/136919)

### CODE REVIEW

**Comment by craftablescience** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> No errors to post.

(Automated response - build 669.)

File: manifests/c/craftablescience/VPKEdit/4.0.0/craftablescience.VPKEdit.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+PackageIdentifier: craftablescience.VPKEdit
+PackageVersion: 4.0.0
+InstallerType: zip
+NestedInstallerType: nullsoft
+NestedInstallerFiles:
+- RelativeFilePath: VPKEdit-4.0.0-win64.exe
+FileExtensions:
+- .bsp
+- .vpk
+Installers:
```
Comment (stephengillie) [inline manifests/c/craftablescience/VPKEdit/4.0.0/craftablescience.VPKEdit.installer.yaml:13]:
```suggestion
Dependencies:
  PackageDependencies:
  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 674.)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**